tests/workflows/parallel_inline_subworkflow_cancellation/tests/test_workflow.py

```diff
@@ -0,0 +1,66 @@
+from vellum.workflows.errors.types import WorkflowErrorCode
+from vellum.workflows.workflows.event_filters import all_workflow_event_filter
+
+from tests.workflows.parallel_inline_subworkflow_cancellation.workflow import (
+    FastFailingNode,
+    ParallelInlineSubworkflowCancellationWorkflow,
+    SlowInlineSubworkflowNode,
+    SlowNode,
+    SlowSubworkflow,
+)
+
+
+def test_parallel_inline_subworkflow_cancellation__streaming():
```

Comment:
DevinAI This test looks perfect. can we now work on the business logic to get this test to pass?